torch/nn/modules/container.py

```diff
@@ -534,7 +534,7 @@ def extra_repr(self) -> str:
                 device_str = '' if not p.is_cuda else ' (GPU {})'.format(p.get_device())
                 parastr = '{} containing: [{} of size {}{}]'.format(
                     "Parameter" if isinstance(p, Parameter) else "Tensor",
-                    torch.typename(p), size_str, device_str)
+                    p.dtype, size_str, device_str)
```

Comment:
This of course gives different results when printing a `ParameterList`, but since the repr isn't a "real repr" anyway, I don't see this being very breaking.